io/zenoh-transport/src/unicast/link.rs

```diff
@@ -284,26 +278,21 @@ async fn rx_task_stream(
             .map_err(|_| zerror!("{}: expired after {} milliseconds", link, lease.as_millis()))??;
         match action {
             Action::Read(n) => {
-                let zs = ZSlice::make(buffer.into(), 0, n)
-                    .map_err(|_| zerror!("{}: decoding error", link))?;
-                zbuf.append(zs);
-
-                let mut zbuf = zbuf.reader();
                 #[cfg(feature = "stats")]
                 transport.stats.inc_rx_bytes(2 + n); // Account for the batch len encoding (16 bits)
```

Comment:
Not introduced by this PR, but just noticed: It would be good not to reproduce errors of C => We probably want to always wrap cfged statements in a block, to avoid mis-cfg-ing something in the future...

Reply:
Fixed.